src/env.rs

```diff
@@ -41,7 +41,11 @@ impl DeltaEnv {
         let pagers = (
             env::var(DELTA_PAGER).ok(),
             env::var(BAT_PAGER).ok(),
```

Comment:
Would you mind getting rid of all explicit mention of `BAT_PAGER`. `get_pager_executable` will look at `BAT_PAGER` itself.